lib/CsvToMarkdown.js

```diff
@@ -20,7 +20,8 @@ function csvToMarkdown(csvContent, delimiter, hasHeader) {
         if (typeof tabularData[i] == "undefined") {
             tabularData[i] = [];
         }
-        var row = e.split(delimiter);
+        var regex = new RegExp('(?!\\B"[^"]*)' + delimiter + '(?![^"]*"\\B)');
```

Comment:
If you would like to remove the left negative lookahead I will happily merge this.